src/ansys/fluent/core/session_solver.py

```diff
@@ -98,10 +98,12 @@ def _build_from_fluent_connection(self, fluent_connection):
         self._settings_root = None
         self._fluent_version = None
         self._lck = threading.Lock()
-        self.svar_service = service_creator("svar").create(
+        self.solution_variable_service = service_creator("svar").create(
```

Comment:
Was `svar_service` originally intended to be public? Or should it have been name `_svar_service`? I would think so, and the clincher seems to be that it is not mentioned in the tests. If this is correct, we can prefix the new name with `_`.

@ajain-work

Reply:
Done. Thank you.